utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -226,6 +226,16 @@ fun File.toSafeUri(): URI {
     return URI("file", "", fileUri.path, fileUri.query, fileUri.fragment)
 }
 
+/**
+ * Return the bytes equal to this [Int] number of mebibytes (MiB).
+ */
+inline val Int.mebibytes get(): Long = this * 1024L * 1024L
+
+/**
+ * Return the bytes equal to this [Int] number of gibibytes (GiB).
+ */
+inline val Int.gibibytes get(): Long = this * 1024L * 1024L * 1204L
```

Comment:
Should the implementation call `mebibytes` internally?

Reply:
I wasn't sure if that's not overdoing it, what would you prefer?

Reply:
I think I'd have a slight tendency towards doing it, as it IMO increases readability (by reducing the operations) and it documents the relation between the units.